test/tools/isoltest.cpp

```diff
@@ -276,6 +285,55 @@ void setupTerminal()
 #endif
 }
 
+fs::path discoverTestPath()
```

Comment:
Since we do also use the same logic in `soltest` (since a few days), is it worth sharing the code in a static lib (avoiding code duplication and accidental differences over the time).